root.tf

```diff
@@ -197,7 +197,7 @@ module "frontend_alb" {
   alb_target_group_port = 9000
   alb_target_type       = "ip"
   certificate_arn       = module.frontend_certificate.certificate_arn
-  health_check_matcher  = "200,303"
+  health_check_matcher  = "308"
```

Comment:
Is it still useful as a healthcheck? I guess it still lets us know that the Play app is up and running?

Could you add a comment explaining why we need to check for an unusual status code?

Reply:
Yeah, Play won't return 308 unless it's running properly. I've added a comment explaining why it's set like this.